jupyter_book/utils.py

```diff
@@ -79,6 +79,10 @@ def _check_url_page(url_page, content_folder_name):
                 url_page
             )
         )
+    if not url_page.startswith(os.sep):
```

Comment:
hmmm, this actually makes me wonder if this *won't* work if people use windows-style `os.sep`. It's building the URLs of each page based off of these values, and I don't know that it'd replace `\` with `/`. I think that's probably OK to address in a different PR but does this seem correct to you too?

Reply:
I was wondering about this. I used `os.sep` here to be consistent with other code in `utils.py`, but I see other places in the code where `'/'` is used explicitly. And the docs don't mention anything about Windows-style paths. I don't have access to a Windows machine to play around with that though.

Reply:
I bet it would mess up, because I think it generates the page URLs directly from the toc.yml file...hmmm, any chance you could open a different issue about this? I don't think we should block your PR on that issue (which is more of a pre-existing design flaw you shouldn't be responsible for fixing :-) )

Reply:
I will do that.